libusb/libusb.h

```diff
@@ -335,6 +335,7 @@ enum libusb_descriptor_type {
 #define LIBUSB_DT_SS_ENDPOINT_COMPANION_SIZE	6
 #define LIBUSB_DT_BOS_SIZE			5
 #define LIBUSB_DT_DEVICE_CAPABILITY_SIZE	3
+#define LIBUSB_DT_INTERFACE_ASSOCIATION_SIZE			8
```

Comment:
Too many tabs, "8" should be aligned with the rest (with tab stop width 8).

Reply:
Ha, to me they all look unaligned because the Xcode project is configured for 4 space tabs.

We really should proceed with https://github.com/libusb/libusb/pull/1443 so that human effort reviewing and fixing trivialities like this can be automated.

Anyway, fixed it.

Reply:
Thanks. I'd recommend using "git diff" before or "git show" after committing, it exposes such whitespace errors easily and clearly and also lets you review other changes and commit messages from another angle. QED GUIs fail miserably on this.